tutorials/quantize_vit/run_vit_b_quant.py

```diff
@@ -2,6 +2,8 @@
 import torchao
 import torchvision.models.vision_transformer as models
 
+from util import benchmark_model, profiler_runner
+
```

Comment:
```suggestion
torch.set_float32_matmul_precision("high")
```

Reply:
Just wondering, the tutorial is in `bf16`. 
```
# Set the model to evaluation mode
model.eval().cuda().to(torch.bfloat16)
```
Does the flag of float32 have any effect? 
Another question I'm curious about, we have `torch.set_float32_matmul_precision`
but it seems we don't have this for other dtype?